### DISCUSSION TITLE
Is UI15 supported?

### DISCUSSION THREAD

**Discussion by codaroma** [arnoudkooi/SN-Utils#442] [Q&A]
All of the UI15 instances I have access to seem to have a problem with commands like /si glidequery. Is UI15 supported? Is it a jQuery version issue or something similar?
![image](https://github.com/arnoudkooi/ServiceNow-Utils/assets/690253/f93ef044-e375-4e4b-9bb8-3c518fde98a9)

**Comment by arnoudkooi** [comment]
It is not really supported, however, I tried to fix this issue in 7.1.5.0. (now available)
Please let me know if it works.
Curious: why do you use UI15?

**Reply by codaroma** [reply]
Yes, works now in UI15. We don't really use UI15. It is just an old instance that we are planning to retire but contains a lot of useful data.
